readme.md

```diff
@@ -570,6 +570,7 @@
 
 ## Higher Education
 
+- [Academic Writing](https://github.com/maehr/awesome-academic-writing) - Open source tools for painless academic writing.
```

Comment:
The word "painless" in the description is subjective. Also, on the list page itself or its contribution guidelines I did not see anything that enforces only open source tools to be added to it.

Reply:
Please rereview my PR